src/controllers/auth.js

```diff
@@ -15,18 +15,14 @@ const users = [
   },
 ]
 
-const getAll = (req, res) => {
-  res.status(200).json({ data: 'sample GET /v1/admin data :p' })
-}
-
 // Validate an existing user and issue a JWT
-function login(req, res) {
+const login = (req, res) => {
   const { email, password } = req.body
-  const user = users.find((u) => {
-    return u.email === email && u.password === password
+  const user = Users.find(() => {
+    return user.email === email && user.password === password
```

Comment:
This function is used in multiple places - do you see some scope of refactoring to make it [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself#:~:text=Don't%20repeat%20yourself%20(DRY,data%20normalization%20to%20avoid%20redundancy.)